client/components/Windows/Help.vue

```diff
@@ -623,6 +623,24 @@
 				</div>
 			</div>
 
+			<div class="help-item">
+				<div class="subject">
+					<code>/ping nick</code>
+				</div>
+				<div class="description">
+					<p>
+						Send a <abbr title="Client-to-client protocol">CTCP</abbr> PING request and
+						calculate delay between the clients. Read more about CTCP PING in this
```

Comment:
except that's not what we are doing...
We don't measure the delay between the clients but from the TL server, via the irc server to the client and back to the TL server

In essence that
```
[thelounge client] --> [thelounge server] __timer starts__ --> [IRC server] --> [victim] --> [thelounge server] __timer stops__
```

The wording here (and on wikipedia for that matter) is simply misleading, as the communication is asymmetrically via the IRC server.

It's already to have it as a gimmick I guess, but at least mention what it does without people having to look at the code ;)